pypeit/extraction.py

```diff
@@ -793,6 +820,16 @@ def __init__(self, sciImg, sobjs_obj, spectrograph, par, caliBrate, objtype, **k
             msgs.error('Unable to set Echelle orders, likely because they were incorrectly '
                        'assigned in the relevant SlitTraceSet.')
 
+
+    @property
+    def nobj_to_extract(self):
```

Comment:
Add docs

Reply:
Done. Docs were already present in the parent methods.